src/LogHelper.ts

```diff
@@ -113,4 +114,83 @@ export class LogHelper {
       }
     });
   }
+
+  // isProduciton needs a value, just not right now.
+  static logUnitsDelta(delta: Units, player: Player, _isProduction: boolean, options?: Units.Options) {
+    if (options?.log !== undefined && options.log === false) {
```

Comment:
Unrelated to overall change but the `?` operator is growing on me. I believe this can all be written `options?.log === false`.